pkg/arvo/sys/vane/gall.hoon

```diff
@@ -343,11 +356,20 @@
         |=  watches-8
         ^-  [bitt boat boar]
         [inbound outbound (~(run by outbound) |=([acked=? =path] nonce=0))]
+      ::
+      ++  spore-9-to-10
+        |=  old=spore-9
+        =-  old(- %10, outstanding -)
+        %-  ~(run by outstanding.old)
+        |=  q=(qeu remote-request-9)
+        %-  ~(gas to *(qeu remote-request))
+        %+  murn  ~(tap to q)
```

Comment:
this could just be a `+skip`

Reply:
I believe I tried, but it didn't nest properly.